include/GL/glui.h

```diff
@@ -581,7 +642,7 @@ class GLUIAPI GLUI_Glut_Window : public GLUI_Node
     void (*glut_mouse_CB)(int,int,int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
```

Comment:
You seem new to git, new to github, and (apparently) new to collaboration.  

The responsibility of a maintainer is to review, refine and ultimately accept or reject proposed changes in the broader interest of the user base and the project.  There are several things that can make that job much more manageable.  PRs that are simple, have a specific purpose and don't make design changes are more likely to sail through without a fuss.  PRs that don't build typically won't be treated with much seriousness.  Most importantly a willingness to collaborate, go the extra mile, refine and iterate are the key ingredients for contributing.